lexico/cli.py

```diff
@@ -75,9 +76,23 @@ def init():
 @lexico.command()
 def view():
     '''Lists all the words present in your dictionary.'''
-    #TODO: Option for file output
     #TODO: More information/table columns
     words = get_words()
     formatted_words = format_words(words)
     display_words = tabulate_words(formatted_words)
     click.echo_via_pager(display_words)
+
+
+@lexico.command()
+def export():
+    '''Exports the current vovabulary into a CSV file.'''
+    words = get_words()
+    formatted_words = format_words_for_export(words)
+    outputFile = open('vocabulary.csv', 'w', newline='')
```

Comment:
You can use the context manager `with` statement here instead of explicitly opening and closing file here.

```
with open(filename, mode, encoding) as f:
    # Perform operations
```

I have another concern regarding the timestamps of `created_at` and `last_lookup`.
Are you converting them to the standard ISO format instead of the current human readable time (such as `2 minutes ago`) displayed by the `view` command?

Reply:
Okay will do that. 
Yes, I'm converting to ISO formats because if I  don't let's say, a user exports the vocabulary. When he sees the same CSV file again after some time, the time displayed will be with respect to the exported time. I mean for example, when he exports it will '2 minutes ago' and when he sees the same file again it will still be '2 minutes ago'.

Reply:
Good!
I will test the PR locally after you make the changes.